integration-tests/integration_test.go

```diff
@@ -666,6 +675,101 @@ func (s *ProcfsScraperTestSuite) TestProcfsScraper() {
 	}
 }
 
+func (s *ProcessListeningOnPortTestSuite) SetupSuite() {
+
+	s.metrics = map[string]float64{}
+	s.executor = NewExecutor()
+	s.StartContainerStats()
+	s.collector = NewCollectorManager(s.executor, s.T().Name())
+
+	s.collector.Env["COLLECTOR_CONFIG"] = `{"logLevel":"debug","turnOffScrape":false,"scrapeInterval":2}`
+	s.collector.Env["ROX_PROCESSES_LISTENING_ON_PORT"] = "true"
+
+	err := s.collector.Setup()
+	s.Require().NoError(err)
+
+	err = s.collector.Launch()
+	s.Require().NoError(err)
+	time.Sleep(10 * time.Second)
+
+	processImage := "quay.io/rhacs-eng/qa:collector-processes-listening-on-ports"
```

Comment:
The image may not be right when COLLECTOR_QA_TAG is set.

Reply:
Done. This is not done for any other image used by the integration tests.

Reply:
We should either:
- add this as a todo to the tech-debt doc, or
- discard the COLLECTOR_QA_TAG notion in this PR, since we are building the image as part of the run. Not sure that the push target is useful either in this case.

What do you think ?

Reply:
I think the push target is useful when changes to the image are made and it needs to be made available. I have used the push target myself.

Reply:
The collector-processes-listening-on-ports image is not built as part of the run.

Reply:
I created a ticket for this https://issues.redhat.com/browse/ROX-13339

Reply:
Thanks for the clarification; I didn't realize that these images were pushed